Client.xcodeproj/project.pbxproj

```diff
@@ -14646,8 +14658,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/SDWebImage/SDWebImage.git";
 			requirement = {
-				kind = exactVersion;
-				version = 5.12.5;
+				kind = upToNextMajorVersion;
```

Comment:
Any specific reason to no longer use exact version here?

Reply:
No specific reason apart from solving the Braze dependency within Core.
Will update to utilize the one that Braze on our current version specifies and double check that everything is fine 💪